.github/workflows/ci-linux.yaml

```diff
@@ -11,6 +11,7 @@ on:
 env:
     PHPUNIT_FLAGS: "-v"
     SYMFONY_PHPUNIT_DIR: "$HOME/symfony-bridge/.phpunit"
+    MAKER_SKIP_MERCURE_TEST: 1
```

Comment:
`ux-turbo` uses `Doctrine\Commom\Util\ClassUtils::class` which no longer exists. As such, 2 `MakeEntity` tests fail that use Mercure. Keep this here && create an maker bundle issue to fix? Or live with (for now) the red x's on all of our tests?

Reply:
Keep this for now. Then we'll fix it in ux-turbo at some point.